kudu/Dockerfile

```diff
@@ -21,6 +21,9 @@ RUN apt-get install -y ssh \
 # SQL Server gem support
 RUN apt-get install -y unixodbc-dev freetds-dev freetds-bin 
 
+# Squash FS support for KuduLite build
```

Comment:
nit: maybe `Squashfs tools for KuduLite build` would be more precise, since this is mainly to get the `mksquashfs` utility.

Reply:
Move it up to Install dependencies, this adds another layer to docker image